packages/aws-cdk/lib/archive.ts

```diff
@@ -4,6 +4,18 @@ import fs = require('fs-extra');
 import glob = require('glob');
 import path = require('path');
 
+const appendFiles = async (directory: string, files: string[], archive: archiver.Archiver): Promise<void> => {
```

Comment:
I'd rather declare this as `async function appendFiles(directory: string, files: string[], archive: archiver.Archiver): Promise<void> { /*...*/ }`

Reply:
tslint tells me that `async function` violates `non-arrow-function` rule if `function` keyword used.

Even so should I do?